scaling_performance/optimizing_storage.adoc

```diff
@@ -171,14 +171,57 @@ to nodes to support a healthy cluster.
 * OpenStack Cinder: OpenStack Cinder tends to be adept in ROX access mode use cases.
 * Databases: Databases (RDBMSs, NoSQL DBs, etc.) tend to perform best with dedicated block storage.
 
-[[choosing-a-docker-graph-driver]]
-== Choosing a Docker Graph Driver
+[[choosing-a-graph-driver]]
+== Choosing a Graph Driver
+
+Container runtimes store images and containers in a graph driver (a pluggable storage
+technology), such as `DeviceMapper` and`Overlay`. Each have
+advantages and disadvantages.
+
+For more information about `Overlay`, including supportability and usage
+caveats, see the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[Red
+Hat Enterprise Linux (RHEL) 7 Release Notes].
+
+.Graph Driver Comparisons
+|===
+|Name |Description |Benefits |Limitations
+
+|Device Mapper loop-lvm
+|Uses the device mapper thin provisioning module (dm-thin-pool) to implement copy
+on write (CoW) snapshots. For each device mapper graph location, thin pool is
```

Comment:
Should _copy on write_ have hyphens? _copy-on-write_ ?? I do not know if this is the case. Just seems like it should (Google search found both hyphenated and non-hyphenated)